content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+---
+reviewers:
+- zparnold
+title: Overview of Cloud Native Security
+content_template: templates/concept
+weight: 1
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Kubernetes Security (and security in general) is an immense topic that has many
+highly interrelated parts. In today's era where open source software is
+integrated into many of the systems that help web applications run,
+we need some overarching concepts to help guide our intuition about how we can
+think about security holistically. This guide will define a thought-framework for
+what we think of when we imagine "Kubernetes Security." The thought framework we selected is completely arbitrary
+and you should only use it if it helps you think about where to secure your software
+stack.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## The 4C's of Cloud Native Security
+Let's start with a diagram that may help you understand how we think about security in layers.
+{{< note >}}
```

Comment:
Is `The 4Cs of Cloud Native Security` an accepted concept by the `cloud native` security community? I am challenging the use of `we`.

Reply:
I'm a member of the CNCF sig-security, while I never claim to speak on behalf of them, I believe that it is accepted.

Reply:
Great, though, do you want to make a note that the CNCF SIG-security endorses or supports ...
It sounded vague, `we think about security in layers`

Reply:
I agree, so let me think about it